tsconfig.json

```diff
@@ -3,11 +3,15 @@
     "noEmit": true,
     "module": "ESNext",
     "target": "ESNext",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
`bundler` is good for apps, but `node16` is better for libraries. that's because bundler is more permissive about things like file endings, which is good for consuming a variety of libraries, but `node16` is stricter, which is better when your library may be consumed by apps with a variety of different settings

Reply:
Yes, but in the end it will be changed by `unbuild`. Do you think it's still worth it?

Reply:
I'm not familiar with `unbuild` or why they would do that. Ideally it shouldn't or there should be an option to configure it

Reply:
`unbuild` imports all the code and puts it in a single file, then minimizes it, so there is no import after it has been compiled. It means that the `moduleResolution` won't have any impact after all.